fusor-ember-cli/app/routes/deployment.js

```diff
@@ -109,10 +109,12 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
         console.log('isSelectedSubscription is');
         console.log(item.get('isSelectedSubscription'));
 
-        if (item.get('isSelectedSubscription')) {
+        if (item.get('qtyToAttach') > 0) {
 
           // POST /customer_portal/consumers/#{CONSUMER['uuid']}/entitlements?pool=#{POOL['id']}&quantity=#{QUANTITY}
           var url = '/customer_portal/consumers/' + consumerUUID + "/entitlements?pool=" + item.get('id') + "&quantity=" + item.get('qtyToAttach');
+          console.log('POST attach subscriptions using following URL');
+          console.log(url);
```

Comment:
Should we be cleaning console logs before committing to master or are these deliberate?